src/constants/themes/types.ts

```diff
@@ -15,7 +15,7 @@ type Typography =
   | typeof primaryTheme['TYPOGRAPHY']
   | typeof secondaryTheme['TYPOGRAPHY'];
 
-export type ThemeColors = valueof<Colors>;
+export type ThemeColors = valueof<Colors> | '';
```

Comment:
Before this PR there were some colors with empty string value, now that we have the final structure it started to throw error in buttons components where `textColor = ''`